sys-apps/ripgrep/ripgrep-0.10.0.recipe

```diff
@@ -0,0 +1,320 @@
+
+CHECKSUM_SHA256_75="fd2be2d6639d0f8fe6cdda291ad456e23629558d466e2789d2c3e9892bda285d"
+SOURCE_URI_75="https://static.crates.io/crates/ucd-util/ucd-util-0.1.1.crate"
+
+CHECKSUM_SHA256_76="882386231c45df4700b275c7ff55b6f3698780a650026380e72dabe76fa46526"
+SOURCE_URI_76="https://static.crates.io/crates/unicode-width/unicode-width-0.1.5.crate"
+
+CHECKSUM_SHA256_77="fc72304796d0818e357ead4e000d19c9c174ab23dc11093ac919054d20a6a7fc"
+SOURCE_URI_77="https://static.crates.io/crates/unicode-xid/unicode-xid-0.1.0.crate"
+
+CHECKSUM_SHA256_78="382810877fe448991dfc7f0dd6e3ae5d58088fd0ea5e35189655f84e6814fa56"
+SOURCE_URI_78="https://static.crates.io/crates/unreachable/unreachable-1.0.0.crate"
+
+CHECKSUM_SHA256_79="fd70f467df6810094968e2fce0ee1bd0e87157aceb026a8c083bcf5e25b9efe4"
+SOURCE_URI_79="https://static.crates.io/crates/utf8-ranges/utf8-ranges-1.0.1.crate"
+
+CHECKSUM_SHA256_80="7716c242968ee87e5542f8021178248f267f295a5c4803beae8b8b7fd9bc6051"
+SOURCE_URI_80="https://static.crates.io/crates/version_check/version_check-0.1.4.crate"
+
+CHECKSUM_SHA256_81="6a02e4885ed3bc0f2de90ea6dd45ebcbb66dacffe03547fadbb0eeae2770887d"
+SOURCE_URI_81="https://static.crates.io/crates/void/void-1.0.2.crate"
+
+CHECKSUM_SHA256_82="af464bc7be7b785c7ac72e266a6b67c4c9070155606f51655a650a6686204e35"
+SOURCE_URI_82="https://static.crates.io/crates/walkdir/walkdir-2.2.5.crate"
+
+CHECKSUM_SHA256_83="773ef9dcc5f24b7d850d0ff101e542ff24c3b090a9768e03ff889fdef41f00fd"
+SOURCE_URI_83="https://static.crates.io/crates/winapi/winapi-0.3.5.crate"
+
+CHECKSUM_SHA256_84="ac3b87c63620426dd9b991e5ce0329eff545bccbbb34f3be09ff6fb6ab51b7b6"
+SOURCE_URI_84="https://static.crates.io/crates/winapi-i686-pc-windows-gnu/winapi-i686-pc-windows-gnu-0.4.0.crate"
+
+CHECKSUM_SHA256_85="afc5508759c5bf4285e61feb862b6083c8480aec864fa17a81fdec6f69b461ab"
+SOURCE_URI_85="https://static.crates.io/crates/winapi-util/winapi-util-0.1.1.crate"
+
+CHECKSUM_SHA256_86="712e227841d057c1ee1cd2fb22fa7e5a5461ae8e48fa2ca79ec42cfc1931183f"
+SOURCE_URI_86="https://static.crates.io/crates/winapi-x86_64-pc-windows-gnu/winapi-x86_64-pc-windows-gnu-0.4.0.crate"
+
+CHECKSUM_SHA256_87="561ed901ae465d6185fa7864d63fbd5720d0ef718366c9a4dc83cf6170d7e9ba"
+SOURCE_URI_87="https://static.crates.io/crates/wincolor/wincolor-1.0.1.crate"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+SECONDARY_ARCHITECTURES="x86"
+fi
+
+PROVIDES="
+	ripgrep$secondaryArchSuffix = $portVersion
+	cmd:rg
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	lib:libxslt$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:a2x
+	cmd:cargo$secondaryArchSuffix
+	cmd:gcc$secondaryArchSuffix
+	"
+
+defineDebugInfoPackage ripgrep$secondaryArchSuffix \
+	$binDir/rg
+
+BUILD()
+{
+	mkdir -p ~/.cargo/registry/src/github.com-1ecc6299db9ec823
+	cp -r /sources-*/* ~/.cargo/registry/src/github.com-1ecc6299db9ec823
```

Comment:
Suggestion:
```diff
+	for i in {2..87}; do
+		eval temp=\$sourceDir$i
-	cp -r /sources-*/* ~/.cargo/registry/src/github.com-1ecc6299db9ec823
+		cp -r "$temp"/* ~/.cargo/registry/src/github.com-1ecc6299db9ec823
+	done
```
Well, this will not fix the issue. It's just a variant that avoids using the hard-coded `/sources-*/`.
BTW, I did not test it.

Reply:
Works great; thanks!